altruedu/faq_altruedu_deutsch.md

```diff
@@ -0,0 +1,42 @@
+**Was ist altruedu?**
+
+Du bist unter 18 Jahre alt und Schüler:in? Dann kommt hier die gute Nachricht: Bei altruedu denken wir zuallererst an DICH und DEINE Bedürfnisse. Anders als andere Plattformen ist altruedu nicht eine abgeschwächte „Kinderversion“ der „richtigen“ Erwachsenenversion. Bei altruedu geht alles von den Kindern bzw. Jugendlichen aus - von DIR! DU bist die Zukunft unserer Welt und an DICH lohnt es sich zuerst zu denken. Deshalb bist DU bei allem, was wir entwickeln, der Ausgangspunkt.
+In Zusammenarbeit mit zahlreichen Expert:innen aus dem Bereich Bildung und Erziehung sowie mit hunderten Schüler:innen konnten wir ein Netzwerk schaffen, das genau auf DICH abgestimmt ist.
+Bei altruedu sind deine Stärken gefragt und deine Schwächen willkommen. Hier findest du Gleichgesinnte, mit denen du dich vernetzen kannst.
+Egal ob für Nachhilfe oder für Hobbys. Bei altruedu findest du eine Community aus Gleichgesinnten (= *aunity*), die sich gegenseitig unterstützt und mit Schul- und Lebensfragen weiterhilft.
```

Comment:
Bei uns findest du eine Community von Altruedus, in der man sich gegenseitig unterstützt und mit Schul- und Lebensfragen weiterhilft.